--- deps/gyp//pylib/gyp/input.py_   2014-05-01 17:20:02.000000000 -0700
+++ deps/gyp//pylib/gyp/input.py    2014-05-01 17:20:07.000000000 -0700
@@ -2489,7 +2489,7 @@
     print('static library %s has several files with the same basename:\n' %
           target + error + 'Some build systems, e.g. MSVC08, '
           'cannot handle that.')
-    raise GypError('Duplicate basenames in sources section, see list above')
+    #raise GypError('Duplicate basenames in sources section, see list above')
 
 
 def ValidateRulesInTarget(target, target_dict, extra_sources_for_rules):
